docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -242,6 +246,29 @@ As highlighted above, the helper is configured to rewrite *local* URLs to an `as
 - [See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/cdn.js)
 - [See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=cdn)
 
+
+### {{moment}}
+
+```handlebars
+{{moment date format}}
+```
+
+Use [moment](https://momentjs.com/) to format dates.
+
+#### Examples
+
+If you encounter an issue with the last day of the month, use one of the following fixes:
+
+```handlebars
+{{moment date format datejs=false}}
+<!-- To disable date.js -->
+```
+
+```handlebars
+{{moment "January 1, 2022" format="YYYY-MM-DD"}}
+<!-- To call moment.js functions -->
+```
+

```

Comment:
are there any examples of calculations? for example, before we brought the helper in-house, this worked: `{{moment "5 hours ago" "MM/DD/YYYY HH:mm"}}` 

Can we check to see if it still does?